docs/modules/ROOT/pages/openshift-service-mesh/howto-ha-controlplane.adoc

```diff
@@ -0,0 +1,42 @@
+= Making a Service Mesh Control Plane highly available
+
+This how-to assumes that you've already xref:openshift-service-mesh/tutorial-bookinfo.adoc#_step_1_creating_the_servicemeshcontrolplane[created a service mesh control plane].
```

Comment:
`you already have`

Reply:
I did some digging on this one, i might be wrong here ...i just don't like the `'ve`